actions/helm-package/action.yaml

```
@@ -0,0 +1,26 @@
+name: Package Helm chart
+description: Package Helm chart and recreate Helm repository index
+inputs:
+  helm-version:
+    description: "The Helm version."
+    required: false
+    default: "v3.10.1"
+  charts-dir:
+    description: "The directory containing the Helm charts."
+    required: false
+    default: "."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Helm
+      uses: azure/setup-helm@v3
+      with:
+        version: "${{ inputs.helm-version }}"
+
+    - name: Helm package
+      run: |
+        helm package .
+        helm repo index . --merge index.yaml
```

In [Quick's case](https://github.com/bakdata/quick/blob/9063577615f8349e2ef47f1e1ebd52660640e327/.github/workflows/update-helm-chart.yml#L72), the index.yaml is in a different directory. So maybe:
```suggestion
        helm repo index . --merge ${{ inputs.index-location }}
```

---

On the other hand, one could also just move the index.yaml into the chart-directory for this action

---

I would prefer keeping the complexity low at this point. we can always extend it later on